src/update-profile/update-profile.service.ts

```diff
@@ -415,4 +391,19 @@ export class UpdateProfileService implements IUpdateProfile {
 			return jsonObj;
 		}
 	}
+
+    getGender(gender: string): string {
+        if(!gender) {
+            return ''
+        } else {
+            if(gender.toLocaleLowerCase() === 'male') {
```

Comment:
Please make the `male` and the `female` texts constants.

Reply:
Done.